src/traces/surface/attributes.js

```diff
@@ -182,6 +206,20 @@ module.exports = {
             min: 0.00,
             max: 5.00,
             dflt: 0.2
+        },
+        vertexnormalsepsilon: {
```

Comment:
OK. Agreed. I won't be the most used attribute nor is it the most obvious ; might as well be verbose. :+1: